packages/strapi-plugin-graphql/services/Resolvers.js

```diff
@@ -398,7 +398,7 @@ module.exports = {
               queryOpts.skip = convertedParams.start;
 
               switch (association.nature) {
-                case 'manyToMany': {
+                case 'manyToMany':
```

Comment:
This braked is here to the scope of `const` variables declared inside the `case`... that, along with that the close bracket is not changing, also introduce an style/indentation issue 🤔 

Also, I thought have seen another PR making a similar modification... let me search...

Reply:
Ok, please try to fix that as soon as possible.